mcchris/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+		<title>mcchris</title>
+
+		<style>
+			body {
+				background-image: url(./img/bg.jpg);
+				color:white;
+				font-family: monospace;
+				margin: 6rem 1rem;
+				font-size: 120%;
+			}
+			h1, h2, h3 {
+				text-align: center;
+			}
+			.flex {
+				display: flex;
+				flex-wrap: wrap;
+				justify-content: space-evenly;
+				margin: 3rem 0;
+			}
+			a img {
+				height: 6rem;
+				width: 6rem;
+				margin: .5rem 0;
+				background-color: white;
+			}
+			a {
+
+			}
+		</style>
+	</head>
+	<body id='home'>
+		<!-- <a href="://mcchris.com">mcchris.com</a> -->
+		<div id='main'>
+			<div>
+				<h1>buy the new album!!</h1>
+				<h2>#mcchrisisgoodmusic</h2>
+				<h3>physical & digital copies</h3>
+				<div class="flex">
+					<a href="https://indiemerch.com/mcchris/item/67788" target="_blank">
+						<img src="./img/gm_1.jpg" alt="blue">
+					</a>
+					<a href="https://indiemerch.com/mcchris/item/67789" target="_blank">
+						<img src="./img/gm_p.jpg" alt="pink">
+					</a>
+					<a href="https://indiemerch.com/mcchris/item/67097" target="_blank">
+						<img src="./img/gm_c.jpg" alt="cd">
+					</a>
+					<a href="https://amazon.com/dp/B07MM8LJ2Q" target="_blank">
+						<img src="./img/amazon_music.jpg" alt="blue">
+					</a>
+					<a href="https://play.google.com/store/music/album?id=Bm2eosxn2brhvyudcarlwd32tmi" target="_blank">
+						<img src="./img/google_play.png" alt="pink">
+					</a>
+					<a href="https://itunes.apple.com/us/album/mcchrisisgoodmusic/1449207030" target="_blank">
+						<img src="./img/apple_music.png" alt="cd">
+					</a>
```

Comment:
Why target blank?

Reply:
the navigation takes the user to a different site